Web API/addToDatabase.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from content.DataBase import alchemy
+
+# Connect to the database
+engine = create_engine('sqlite:///../content/DataBase/mydatabase.db')
+Session = sessionmaker(bind=engine)
+session = Session()
+
+def add_user(email, id):
```

Comment:
Write documentation any of the standart python documentation formats, here is an example of one -
```python
def count_hi_v4(word: str) -> int:
"""
The function counts how many times the string "hi" appears in a given string.
@param word: The given string.
@return: The amount of times the word "hi" appeared.
"""
return word.count("hi")
```